src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -34,36 +34,27 @@ export class ArtistAuctionResultItem extends Component<Props> {
         {({ user, mediator }) => {
           return (
             <Row>
-              <Responsive>
-                {({ xs, sm, md }) => {
-                  if (xs) {
-                    return (
-                      <ExtraSmallAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  } else if (sm || md) {
-                    return (
-                      <SmallAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  } else {
-                    return (
-                      <LargeAuctionItem
-                        {...this.props}
-                        mediator={mediator}
-                        user={user}
-                      />
-                    )
-                  }
-                }}
-              </Responsive>
-
+              <Media at="xs">
+                <ExtraSmallAuctionItem
+                  {...this.props}
+                  mediator={mediator}
+                  user={user}
+                />
+              </Media>
+              <Media between={["sm", "lg"]}>
```

Comment:
This makes me think -- would an expanded `at` make sense here (can be string or array)? Something like: 

```js
<Media at={['sm', 'md']}>...
```

From a design handoff to dev layout perspective i feel like this is (slightly) easier to think about.

Reply:
Oh yeah, great point, I like that! 👌

Reply:
That's in addition to between? It could be useful if we had non-contiguous designs. I'm not sure how likely that is though.

Reply:
Yeah I was thinking of leaving `between` for now. It’s already written and seems likely to be useful.

Reply:
I think that `at={['sm', 'md']}` is easier to reason about since it maps 1:1 with our design assets (and thus will be more friendly over time), but that `between` should remain in the underlying API since its a powerful building block.

Reply:
<img width="781" alt="screen shot 2018-10-10 at 1 51 23 pm" src="https://user-images.githubusercontent.com/236943/46765340-9b25ba80-cc93-11e8-9dd5-083d60c6237d.png">

Reply:
It seems a bit weird to have to mention XL in our code when thinking specifically about L and M

Reply:
I imagine `between` being useful when programatically using the API.

Reply:
I definitely think we should leave `between`. I already ran across a usecase for `at` though... (or `notAt`?) 

https://github.com/artsy/reaction/pull/1379/files#diff-17af9a6729cede275ff0d1e7714fdc5bR130

That would be `<Media at={["xs", "lg", "xl"]}/>`

Reply:
Meh, I read it wrong. That's only `lg` and `xl`. Component is complicated 😓

Reply:
I’m going to defer this until a next iteration, as it will require some (required anyways) refactoring first.